src/Datadog.Trace/StatusCode.cs

```diff
@@ -0,0 +1,23 @@
+namespace Datadog.Trace
```

Comment:
Taken from https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry.Api/Trace/StatusCode.cs

Reply:
maybe I should mention it in comment?

Reply:
done